onto_tool/onto_tool.py

```diff
@@ -194,8 +194,14 @@ def addDefinedBy(g, ontologyIRI):
             owl:ObjectProperty
             owl:DatatypeProperty
             owl:AnnotationProperty
+            owl:Thing
+          }
+          ?defined a ?dtype .
+          FILTER(!ISBLANK(?defined))
+          FILTER EXISTS {
+            ?defined ?anotherProp ?value .
+            FILTER (?anotherProp != rdf:type)
```

Comment:
@rjyounes This is the key part, adds the extra annotations to `owl:Thing`s, but only non-stub ones.